app/src/main/res/layout/activity_album_review.xml

```diff
@@ -0,0 +1,93 @@
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:weightSum="1"
+    android:background="@drawable/my_recycleview_element_border"
+    android:padding="@dimen/album_art_border">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/album_art_size"
+        android:orientation="vertical"
+        android:background="@drawable/my_recycleview_element_border"
+        android:padding="@dimen/album_info_border">
+
+        <TextView
+            android:id="@+id/album_name_review"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_weight="0.5"
+            android:background="@color/colorPrimary"
+            android:gravity="center"
+            android:text="36 Chambers"
+            android:textColor="@android:color/black"
+            android:textSize="@dimen/list_font_size"
+            android:textStyle="bold"
+            android:padding="@dimen/album_art_border"/>
+
+        <TextView
+            android:id="@+id/artist_name_review"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_gravity="left"
+            android:layout_weight="0.5"
+            android:background="@color/colorPrimary"
+            android:gravity="center"
+            android:text="Wu Tang Clan"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/list_font_size"
+            android:textStyle="bold" />
+    </LinearLayout>
+
+    <ImageView
+        android:id="@+id/album_art_review"
+        android:layout_height="@dimen/album_art_size"
+        android:layout_width="@dimen/album_art_size"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_centerVertical="true"
+        android:src="@mipmap/ic_launcher"
+        android:layout_gravity="center" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:background="@drawable/my_recycleview_element_border"
+        android:padding="@dimen/album_info_border">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:background="@color/colorPrimary">
+            <RatingBar
+                android:layout_below="@+id/rate_me"
+                android:id="@+id/ratingBar"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:numStars="5"
+                android:stepSize="1.0"
+                android:layout_gravity="center"
+                android:background="@color/colorPrimary" />
+        </LinearLayout>
+    </LinearLayout>
+
+    <LinearLayout
```

Comment:
Sorry it does have a child but I don't think the LinearLayout is necessary.

Remove the LinearLayout and change the EditText to:

<EditText
    android:id="@+id/editText"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:hint="@string/review_hint"
    android:gravity="top|left"
    android:background="@color/colorPrimary"/>